magda-web-client/src/Components/Dataset/Add/DatasetAddFilesPage.tsx

```diff
@@ -90,6 +90,7 @@ class DatasetAddFilesPage extends React.Component<
                 // --- skip the directory item
                 continue;
             }
+            if (!thisFile) continue;
```

Comment:
What's this for? Shouldn't it be unreachable because of the if statement above?

Reply:
Yeah good catch. It was a remnant from a previous PR and a result of merge somewhere down the line, I think :man_shrugging: